### PR TITLE
Bladder documentation

### PR DESCRIPTION
**Pull request by zekh167**
Hi Richard. I added the documentation for bladder and added the image as well.

### CODE REVIEW

File: docs/scaffolds/bladder.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+As with the other organ systems, the lower urinary tract (LUT) contains different sub-scaffolds consisting of the bladder, its ureters, and the urethra.
+
+The bladder scaffold is a 3-D volumetric model, designed topologically as a hollow organ defined by a central line which follows the path of a bladder traced from an image.
+
+The current scaffold consists of different parts to represent the bladder body, bladder neck, ureters opening on the bladder, and the urethra. The neck of the bladder is represented in the scaffold as a narrowing of the body at the region where ureters penetrate the surface into the lumen of the bladder.
+
+.. note::
+
+   Separate scaffold will be provided for the bladder.
```
Comment (rchristie) [inline docs/scaffolds/bladder.rst:22]: A separate bladder-only scaffold is in development.

File: docs/scaffolds/bladder.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Variants
+--------
+
+The bladder scaffold is provided with parameter sets for the following five species, which are very similar in nature but have small, intricate differences:
```
Comment (rchristie) [inline docs/scaffolds/bladder.rst:27]: , which differ subtly in shape and location of ureter inlets.

File: docs/scaffolds/bladder.rst (hunk 1 above)
Comment (rchristie) [inline docs/scaffolds/bladder.rst:22]: A separate bladder-only scaffold is in development.

File: docs/scaffolds/bladder.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Coordinates
+-----------
+
+The bladder scaffold defines the geometric and flat coordinates.
+
+The geometric ``coordinates`` field gives an approximate, idealized representation of the bladder shape for the species, which is intended to be fitted to actual data for a specimen.
+
+The ``flat coordinates`` represents the geometric field when the bladder scaffold is cut along its length and laid flat. This field is intended for fitting data obtained from a flat bladder preparation.
+
+A material coordinates field is not provided, so to perform embedding at this time, it is necessary to use the generic ``coordinates`` field as material coordinates.
```
Comment (rchristie) [inline docs/scaffolds/bladder.rst:52]: Add: This omission will be fixed in the future bladder-only scaffold.

File: docs/scaffolds/bladder.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+* Cat
+* Human
+* Mouse
+* Pig
+* Rat
+
+These variants' geometry and annotations are best viewed in the **Scaffold Creator** tool in the ABI Mapping Tools. On the web, the latest published generic bladder scaffold variants can be viewed on the `SPARC Portal <https://sparc.science/>`_ by searching for ``bladder``, filtering for models, selecting a variant and viewing the scaffold in its Gallery tab.
+
+The bladder scaffold script generates the scaffold mesh and geometry based on some parameters include the diameters for different axes, wall thickness, neck angle (for bladder only), etc., which are all defined for the bladder and urethra separately. The ability to control these parameters, as well as variations of them throughout the length of the bladder and urethra allows the scaffold to be configurable for different species and individuals.
+
```
Comment (rchristie) [inline docs/scaffolds/bladder.rst:38]: Add as separate paragraph: The urethra is optional and can be turned on with the *Include urethra* setting. By default the ureter inlets are marker points on the inner (luminal) surface, but by checking the *Include ureters* option these can be made as 3D element structures.

**Review by rchristie — CHANGES_REQUESTED** [review]
Looks good, some small improvements in comments.

**Comment by zekh167** [comment]
Thanks Richard. I applied your comments.

**Review by rchristie — APPROVED** [review]
Great, thanks!

**Pull request merged**